tests/basic/pure_test.go

```diff
@@ -115,3 +116,41 @@ var _ = Describe("{PureVolumeCRUDWithPXCTL}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+// This test performs a basic test making sure Volume mount options are working as expected.
+var _ = Describe("{PureFaFbValidateVolumemount}", func() {
```

Comment:
We should add this to the existing test, ideally doing it a bit more dynamically by checking what mount options the storage class provides and then checking for those in the mount. That way we avoid adding an extra test for every specific function we want to check

Reply:
Done